src/cf-forms.less

```diff
@@ -197,26 +217,30 @@ textarea {
 
 .@{cf-forms_input-icon-class} {
     @font-size: 20px;
-
-    position: relative;
-    top: unit(6px / @font-size, em);
-    margin-left: unit(4px / @font-size, em);
     font-size: unit(@font-size / @base-font-size-px, em);
 }
 
+.form-label-error .@{cf-forms_input-icon-class} {
+    color: @input-error;
+}
+
+// Deprecated
```

Comment:
Yeah, you can hack it by doing something like:

    // Deprecated
     .warning + .@{cf-forms_input-icon-class} {
         color: `(function(){
           console.log('cf-forms warning is deprecated!');
           return @{input-warning};
         })()`;
     }